src/Middleware/OutputCaching/src/LoggerExtensions.cs

```diff
@@ -0,0 +1,128 @@
+
+    [LoggerMessage(5, LogLevel.Debug, "Adding a minimum freshness requirement of {Duration} specified by the 'min-fresh' cache directive.",
+        EventName = "LogRequestMethodNotCacheable")]
+    internal static partial void ExpirationMinFreshAdded(this ILogger logger, TimeSpan duration);
+
+    [LoggerMessage(6, LogLevel.Debug, "The age of the entry is {Age} and has exceeded the maximum age for shared caches of {SharedMaxAge} specified by the 's-maxage' cache directive.",
+        EventName = "ExpirationSharedMaxAgeExceeded")]
+    internal static partial void ExpirationSharedMaxAgeExceeded(this ILogger logger, TimeSpan age, TimeSpan sharedMaxAge);
+
+    [LoggerMessage(7, LogLevel.Debug, "The age of the entry is {Age} and has exceeded the maximum age of {MaxAge} specified by the 'max-age' cache directive. " +
+        "It must be revalidated because the 'must-revalidate' or 'proxy-revalidate' cache directive is specified.",
+        EventName = "ExpirationMustRevalidate")]
+    internal static partial void ExpirationMustRevalidate(this ILogger logger, TimeSpan age, TimeSpan maxAge);
+
+    [LoggerMessage(8, LogLevel.Debug, "The age of the entry is {Age} and has exceeded the maximum age of {MaxAge} specified by the 'max-age' cache directive. " +
+        "However, it satisfied the maximum stale allowance of {MaxStale} specified by the 'max-stale' cache directive.",
+        EventName = "ExpirationMaxStaleSatisfied")]
+    internal static partial void ExpirationMaxStaleSatisfied(this ILogger logger, TimeSpan age, TimeSpan maxAge, TimeSpan maxStale);
+
+    [LoggerMessage(9, LogLevel.Debug, "The age of the entry is {Age} and has exceeded the maximum age of {MaxAge} specified by the 'max-age' cache directive.", EventName = "ExpirationMaxAgeExceeded")]
+    internal static partial void ExpirationMaxAgeExceeded(this ILogger logger, TimeSpan age, TimeSpan maxAge);
+
+    [LoggerMessage(10, LogLevel.Debug, "The response time of the entry is {ResponseTime} and has exceeded its expiry date of {Expired} specified by the 'Expires' header.",
+        EventName = "ExpirationExpiresExceeded")]
+    internal static partial void ExpirationExpiresExceeded(this ILogger logger, DateTimeOffset responseTime, DateTimeOffset expired);
+
+    [LoggerMessage(11, LogLevel.Debug, "Response is not cacheable because it does not contain the 'public' cache directive.",
+        EventName = "ResponseWithoutPublicNotCacheable")]
+    internal static partial void ResponseWithoutPublicNotCacheable(this ILogger logger);
+
+    [LoggerMessage(12, LogLevel.Debug, "Response is not cacheable because it or its corresponding request contains a 'no-store' cache directive.",
+        EventName = "ResponseWithNoStoreNotCacheable")]
+    internal static partial void ResponseWithNoStoreNotCacheable(this ILogger logger);
+    [LoggerMessage(13, LogLevel.Debug, "Response is not cacheable because it contains a 'no-cache' cache directive.",
```

Comment:
```suggestion
    internal static partial void ResponseWithNoStoreNotCacheable(this ILogger logger);

    [LoggerMessage(13, LogLevel.Debug, "Response is not cacheable because it contains a 'no-cache' cache directive.",
```

Reply:
I will need to remove all the unused messages that were copied from Response caching.